report/extra_metadata_templates.go

```diff
@@ -0,0 +1,33 @@
+package report
+
+type ExtraMetadataTemplates []MetadataRowTemplate
```

Comment:
> I was debating adding a "sortable" priority column

Yes, add a order field which defines how templates should be sorted.  Then use a map.

> commutative. Any other alternatives?

I don't think the commutative merge needs to be particularly useful on conflict, just well defined.  It might be enough to take the 'biggest' template on id conflict, where biggest is defined lexicographically on all the fields.